app/src/main/res/xml/preferences.xml

```diff
@@ -64,7 +67,8 @@
 
     <androidx.preference.PreferenceCategory
         android:title="@string/preferences_category_privacy_security"
-        app:iconSpaceReserved="false">
+        app:iconSpaceReserved="false"
+        android:layout="@layout/preference_category">
```

Comment:
It's going to be easy to forget to add this when adding new `PreferenceCategory`s, so I don't love this solution, but the `Preference` API is rough like that sometimes.  If you haven't found any better solutions, we'll have to live with this.